data/base-spells.yaml

```diff
@@ -227,6 +227,12 @@ spell_data:
     ability: Targeted Magic
     abbrev: IZ
     cyclic: true
+  Mind Shout:
+    ability: Debilitation
+    abbrev: MS
+    mana: 20
```

Comment:
It is the minimum prep though, if merge works the way it is documented (and how I understand it) then user provided settings would override this